package.json

```diff
@@ -41,41 +41,41 @@
     "@commitlint/config-conventional": "^7.0.1",
     "@commitlint/config-lerna-scopes": "^7.0.0",
     "@commitlint/travis-cli": "^7.0.0",
-    "babel-core": "^6.26.0",
+    "babel-core": "^6.26.3",
     "babel-plugin-external-helpers": "^6.22.0",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.6.1",
+    "babel-preset-env": "^1.7.0",
     "chai": "^4.1.2",
-    "deepmerge": "^2.1.0",
+    "deepmerge": "^2.1.1",
     "docsify-cli": "^4.2.1",
     "eslint": "^4.19.1",
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-node": "^5.2.1",
-    "eslint-plugin-prettier": "^2.6.0",
+    "eslint-plugin-prettier": "^2.6.2",
     "formdata-polyfill": "^3.0.11",
     "gh-pages": "^1.2.0",
     "har-validator": "^5.1.0",
     "husky": "^0.14.3",
-    "jest": "^23.4.1",
+    "jest": "^23.5.0",
     "jest-tap-reporter": "^1.9.0",
     "lerna": "^2.11.0",
```

Comment:
lerna cannot be bumped yet to 3.0.0 since commitlint lerna scopes currently doesn't work with it.  Someone has a PR opened to address it.